reconcile/gql_definitions/vault_instances/vault_instances.py

```diff
@@ -0,0 +1,234 @@
+"""
+Generated by qenerate plugin=pydantic_v1. DO NOT MODIFY MANUALLY!
+"""
+from enum import Enum  # noqa: F401 # pylint: disable=W0611
+from typing import (  # noqa: F401 # pylint: disable=W0611
+    Any,
+    Callable,
```

Comment:
you will need to regenerate your classes with the latest qenerate to make sure this is imported from `collections.abc`

Reply:
Done!